### DISCUSSION TITLE
Is it possible to create a background-like app that sits in the menubar?

### DISCUSSION THREAD

**Discussion by extempl** [wailsapp/wails#2614] [Q&A] [answered]
Like Harvest as an example: 
![](https://dsh.re/4fffa)

I tried to find something related to this in the docs, but has no luck. I guess, Wails is good only for detached window, am I right?

**Comment by leaanthony** [comment, marked as answer]
For v2, yes. For v3, we will have this support. Not sure the window will be a popover like the above but we could add methods to centre it to the icon?
